### PR TITLE
Fixes #25

### LINKED ISSUE #25: Internal Server Error
`MAX_WAIT_TIME` is undefined in `index.js`

The definition is only made in `wait.js`.

`res.status().json` is not a function.

`res.json` is sufficient

### PR DESCRIPTION
**Pull request by aulisius**
Fixes #25 

* Uses destructuring to initialize env variables
* `res.json` takes a status parameter.

### CODE REVIEW

File: src/wait.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const MAX_WAIT_TIME = process.env.MAX_WAIT_TIME || 90000;
+const { MAX_WAIT_TIME = 90000 } = process.env;
```
Comment (michaelsanford) [inline src/wait.js:1]: Nice use of destructuring.

**Review by michaelsanford — APPROVED** [review]
Thanks for the ES6 updates, too.

**Comment by michaelsanford** [comment]
Thanks for your contribution! 👏

**Comment by aulisius** [comment]
Thanks for accepting the PR! :D

**Pull request merged**